src/main/java/frc/robot/OI.java

```diff
@@ -70,7 +70,7 @@ public static void setRumble(double rumbleIntensity) {
   }
 
   public static final class Operator {
-    private static final XboxController controller = new XboxController(operatorJoystickPort);
+    public static final XboxController controller = new XboxController(operatorJoystickPort);
```

Comment:
Talked with @KianChilds about this.  Reason is the getPOV is not implemented in OI.  He will open an issue and reference here for us to fix after next competition